adaptive-spec-decode/scripts/benchmark_baseline.py

```
@@ -0,0 +1,92 @@
+
+def free_gpu():
+    gc.collect()
+    torch.cuda.empty_cache()
+
+def main():
+    # VANILLA
+    print("Loading vanilla model...")
+    llm = LLM(model=TARGET, gpu_memory_utilization=0.90,
+              max_model_len=2048, enforce_eager=True)
+    vanilla = run_bench("VANILLA", llm)
+    del llm; free_gpu()
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)